common/web/gesture-recognizer/src/engine/headless/gestures/specs/pathModel.ts

```diff
@@ -0,0 +1,15 @@
+import { GesturePath } from "../../gesturePath.js";
+
+// The TrackedPath model only cares about if the path matches... not what that MEANS.
+// THAT is the role of the gesture Model (model.ts).
+
+export interface PathModel {
+  /**
+   * Given a TrackedPath, indicates whether or not the path matches this PathModel.
+   *
+   * May return null or undefined to signal 'continue'.
+   *
+   * @param path
+   */
+  evaluate(path: GesturePath<any>): 'reject' | 'resolve' | undefined;
```

Comment:
Is this return value just coincidentally the same as the ContactModel.onItemChange return value? Or should we be extracting the type?

I wonder if it is better also to be explicit about the return value: `'reject' | 'resolve' | 'continue'` and disallow `undefined`/`null` in both places?